bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -123,6 +101,12 @@ pub enum ExecState {
     EndBlock,
 }
 
+impl Default for ExecState {
+    fn default() -> Self {
+        ExecState::Op(OpcodeId::INVALID(0))
+    }
```

Comment:
i know this code is from the old ExecStep::default().  But INVALID(0) is quite anti intuition. I would suggest 

(1) change to OpcodeId::INVALID(0xef)
(2) change to OpcodeId::STOP ( it is 0x0). // i would prefer this 

```
ExecStep
```